src/testRunner/unittests/tsbuild.ts

```diff
@@ -467,84 +467,225 @@ export const b = new A();`);
-                    "/src/2/second-output.js.map",
-                    "/src/2/second-output.d.ts",
-                    "/src/2/second-output.d.ts.map"
-                ];
 
-                assert.equal(actualReadFileMap.size, expected.length, `Expected: ${JSON.stringify(expected)} \nActual: ${JSON.stringify(arrayFrom(actualReadFileMap.entries()))}`);
-                expected.forEach(expectedValue => {
-                    const actual = actualReadFileMap.get(expectedValue);
-                    assert.equal(actual, 1, `Mismatch in read file call number for: ${expectedValue}\nExpected: ${JSON.stringify(expected)} \nActual: ${JSON.stringify(arrayFrom(actualReadFileMap.entries()))}`);
+                    const patch = fs!.diff();
+                    // tslint:disable-next-line:no-null-keyword
+                    Harness.Baseline.runBaseline(`outFile-${scenario.split(" ").join("-")}.js`, patch ? vfs.formatPatch(patch) : null);
+                });
+                it("verify readFile calls", () => {
+                    const expected = [
+                        // Configs
+                        "/src/third/tsconfig.json",
+                        "/src/second/tsconfig.json",
+                        "/src/first/tsconfig.json",
+
+                        // Source files
+                        "/src/third/third_part1.ts",
+                        "/src/second/second_part1.ts",
+                        "/src/second/second_part2.ts",
+                        "/src/first/first_PART1.ts",
+                        "/src/first/first_part2.ts",
+                        "/src/first/first_part3.ts",
+
+                        // Additional source Files
+                        ...(additionalSourceFiles || emptyArray),
+
+
+                        // outputs
+                        "/src/first/bin/first-output.js",
+                        "/src/first/bin/first-output.js.map",
+                        "/src/first/bin/first-output.d.ts",
+                        "/src/first/bin/first-output.d.ts.map",
+                        "/src/2/second-output.js",
+                        "/src/2/second-output.js.map",
+                        "/src/2/second-output.d.ts",
+                        "/src/2/second-output.d.ts.map"
+                    ];
+
+                    assert.equal(actualReadFileMap.size, expected.length, `Expected: ${JSON.stringify(expected)} \nActual: ${JSON.stringify(arrayFrom(actualReadFileMap.entries()))}`);
+                    expected.forEach(expectedValue => {
+                        const actual = actualReadFileMap.get(expectedValue);
+                        assert.equal(actual, 1, `Mismatch in read file call number for: ${expectedValue}\nExpected: ${JSON.stringify(expected)} \nActual: ${JSON.stringify(arrayFrom(actualReadFileMap.entries()))}`);
+                    });
                 });
             });
+        }
+
+        verifyOutFileScenario("baseline sectioned sourcemaps", noop);

```

Comment:
Should this `noop` test still be here?

Reply:
Ohhh, the `noop` cb isn't for the test content, it's for a modification to the base test content, got it.